ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -535,6 +557,19 @@ interface Ethernet28
 
 # Filters
 
+# DOT1X
+
+## DOT1X Summary
+
+### DOT1X Interfaces
+
+| Interface | State | Phone Force Authorized |
+| --------- | ----- | ----------- |
+| Ethernet29 | auto | True |
+| Ethernet30 | force-authorized | False |
+| Ethernet31 | force-unauthorized | - |
+
+
```

Comment:
double blank lines. Caused by wrong placement in templates. I will highlight there.

Reply:
completed.